cover_my_meds.gemspec

```diff
@@ -12,26 +10,23 @@ Gem::Specification.new do |spec|
   spec.summary       = %q{CoverMyMeds Public API}
   spec.description   = %q{The public version of CoverMyMeds API}
   spec.homepage      = "https://github.com/covermymeds/cover_my_meds"
-
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
```

Comment:
There are no executables, and lib is already the default require path. No idea why the default name was even changed.